pwa/src/templates/templateParts/header/HeaderTemplate.tsx

```diff
@@ -12,6 +12,7 @@ import { ExternalLinkIcon } from "@gemeente-denhaag/icons";
 import clsx from "clsx";
```

Comment:
The code is fine like this.

The sub-pages _without_ the header look much better like this as well.

However, I do feel as if the header is very redundant at this point (a kind of redundant title with a kind of redundant subtitle, not adding much value).

What do you think of something like this:

![image](https://user-images.githubusercontent.com/66552932/191227247-a0acfbf6-a62c-4fa8-b685-7684d547b59c.png)